### ISSUE TITLE
MenuItem: iconClass is ignored if MenuItem has a routerLink

### ISSUE THREAD

**Issue by ijager** [primefaces/primeng#13040]
### Describe the bug

```ts
items: MenuItem[] = [
    {
      label: "User",
      icon: "bi bi-person-circle",
      iconClass: 'text-3xl',
      command: () => this.goToUserPage(),
    },
    {
      label: "System Information",
      icon: "bi bi-info-circle-fill",
      iconClass: 'text-3xl',
      routerLink: ["/", "info"]
    },
```

The iconClass `'text-3xl'` is applied for the first item `User` but not for the second item. However if I remove the routerLink then the class is applied and the icon becomes bigger.

### Environment

Primeng 15.2.0

### Angular version

15.1.4

### PrimeNG version

15.2.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.19.0

### Browser(s)

Safari

### Steps to reproduce the behavior

Add a routerLink to a `MenuItem` with an `iconClass` set. Then notice how the iconClass is not applied.

### Expected behavior

I expect that the iconClass is still applied inside a `router-link` element.

**Comment by volvachev** [comment]
Hi, this is fixed in #13210 pr

**Comment by cetincakiroglu** [comment]
Closing the issue since it's fixed in #13210 
Please re-open if the problem persists.
